common/reactive/src/main/java/io/helidon/common/reactive/OutputStreamPublisher.java

```diff
@@ -163,6 +195,7 @@ private void complete(Throwable t) {
         subscriber.close(sub -> {
             synchronized (invocationLock) {
                 sub.onError(t);
```

Comment:
There is a race condition against `requested.increment(n, t -> complete(t));` on line 57(not caused by this PR), can be solved by removing synchronization logic from this publisher and wrapping subscriber in `io.helidon.common.reactive.SequentialSubscriber`